# HG changeset patch
# User jwe
# Date 745019753 0
#      Tue Aug 10 21:55:53 1993 +0000
# Node ID e276a012f7dba13ba42626a17c6520f99920b8c5
# Parent  82044b0a4b5fded5870b8b20cf6a03e534864e28
[project @ 1993-08-10 21:55:53 by jwe]
Include f-balanc.h and f-givens.h.

diff --git a/src/g-builtins.cc b/src/g-builtins.cc
--- a/src/g-builtins.cc
+++ b/src/g-builtins.cc
@@ -41,23 +41,25 @@ Software Foundation, Inc.
 #include <iostream.h>
 #include <fstream.h>
 #include <stdio.h>
 #include <sys/wait.h>
 #include <sys/param.h>
 #include <signal.h>
 #include <math.h>
 
+#include "f-balance.h"
 #include "f-colloc.h"
 #include "f-dassl.h"
 #include "f-det.h"
 #include "f-eig.h"
 #include "f-fft.h"
 #include "f-fsolve.h"
 #include "f-fsqp.h"
+#include "f-givens.h"
 #include "f-hess.h"
 #include "f-ifft.h"
 #include "f-inv.h"
 #include "f-lpsolve.h"
 #include "f-lsode.h"
 #include "f-lu.h"
 #include "f-npsol.h"
 #include "f-qpsol.h"
